gatsby/src/pages/borderline-substances/{BnfBorderlineSubstancesTaxonomy.slug}.tsx

```diff
@@ -0,0 +1,138 @@
+import { graphql, Link } from "gatsby";
+import React, { type FC } from "react";
+
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import { PageHeader } from "@nice-digital/nds-page-header";
+
+import { BorderlineSubstancesMenu } from "@/components/BorderlineSubstancesMenu/BorderlineSubstancesMenu";
+import { Layout } from "@/components/Layout/Layout";
+import { SEO } from "@/components/SEO/SEO";
+import { useSiteMetadata } from "@/hooks/useSiteMetadata";
+import { type SlugAndTitle } from "@/utils";
+
+import styles from "./{BnfBorderlineSubstancesTaxonomy.slug}.module.scss";
+
+export type BorderlineSubstancesSectionPageProps = {
+	data: {
+		bnfBorderlineSubstancesTaxonomy: SlugAndTitle & {
+			rootTaxonomy: {
+				childTaxonomies: {
+					title: string;
+					childTaxonomies: {
+						title: string;
+						slug: string;
+					}[];
+					slug: string;
+				}[];
+			};
+		};
+	};
+};
+
+const BorderlineSubstancesSectionPage: FC<
+	BorderlineSubstancesSectionPageProps
+> = ({
+	data: {
+		bnfBorderlineSubstancesTaxonomy: { slug, title, rootTaxonomy },
+	},
+}) => {
+	const { siteTitleShort } = useSiteMetadata();
+	const sections = rootTaxonomy.childTaxonomies;
+	const isOneLevel = sections.find(
+		(section) =>
+			section.childTaxonomies != null && section.childTaxonomies.length > 0
+	)
+		? false
+		: true;
+
+	return (
+		<Layout>
+			<SEO title={title} />
+
+			<Breadcrumbs>
+				<Breadcrumb key="NICE" to="https://www.nice.org.uk/">
```

Comment:
I don't think you need `key` attributes on these breadcrumbs. They're only need if you're putting them in an array